README.md

```diff
@@ -1,26 +1,90 @@
-#**Launching tests:**
+#**How search works**
 
-1. Create databases:\
-    `CREATE DATABASE athena_db_test OWNER ohdsi;`\
-    `GRANT ALL PRIVILEGES ON DATABASE athena_db_test TO ohdsi;`
+There are 2 modes of search: exact and non-exact. Both are case insensitive.
+
+### Exact mode
+For exact mode you need to enclose your phrase or single word in quotes like this:
+1. `"quillaia liquid extract"`
+2. `"aspirin"`\
+You can combine exact and non-exact words in one phrase:\
+`ibuprofen "quillaia liquid extract"`
+### Non-exact mode
+Brackets "[]", braces "{}" and parentheses "()" are ignored. Results for queries `aspirin` and `(aspirin)` will be the same.\
+This kind of search is "fuzzy" which means that synonyms are suggested.\
+Example of a query with a typo:\
+        `asppirin allergy`\
+Results are shown in relevance-descending order:\
+![](src/test/resources/examples/search_with_a_typo.png)
+Example of a query with partial concept code ("2" is missing):\
+![](src/test/resources/examples/part_concept_code.png)
+        
+#### Example of SOLR query:
+            
+            concept_name_ci:aspirin^9 OR 
```

Comment:
Is there a reason for purely tech thing in readme? Maybe it makes sense to remove it from here?